src/WebServer.ino

```diff
@@ -3798,9 +3800,70 @@ void handle_json()
   }
   if (!showSpecificTask) {
     TXBuffer += F("],\n");
-    stream_last_json_object_value(F("TTL"), String(ttl_json * 1000));
+    stream_next_json_object_value(F("TTL"), String(ttl_json * 1000));
   }
 
+  if(showNodes) {
+    bool comma_between=false;
+    for (byte x = 0; x < UNIT_MAX; x++)
+    {
+      if (Nodes[x].ip[0] != 0)
+      {
+
+        char ip[20];
+
+        sprintf_P(ip, PSTR("%u.%u.%u.%u"), Nodes[x].ip[0], Nodes[x].ip[1], Nodes[x].ip[2], Nodes[x].ip[3]);
+
+        if( comma_between ) {
+          TXBuffer += F(",");
+        } else {
+          comma_between=true;
+          TXBuffer += F("\"nodes\":[\n"); // open json array if >0 nodes
+        }
+
+        TXBuffer += F("{");
+        stream_next_json_object_value(F("nr"), String(x));
+
+        if (x != Settings.Unit) {
+          stream_next_json_object_value(F("name"), Nodes[x].nodeName);
+        } else {
+          stream_next_json_object_value(F("name"), Settings.Name);
+        }
+
+        if (Nodes[x].build) {
+          stream_next_json_object_value(F("build"), String(Nodes[x].build));
+        }
+
+        if (Nodes[x].nodeType) {
+          switch (Nodes[x].nodeType)
+          {
+            case NODE_TYPE_ID_ESP_EASY_STD:
+              stream_next_json_object_value(F("type"), F("ESP Easy"));
```

Comment:
It is better to have the `F("type")` stored in a temporary `String`, since those F() macro strings will appear multiple times in the bin file, if done like this.
Also I am not sure "type" is the best description for this. I guess "platform" is a better choice.